### ISSUE TITLE
vllm开启后，合成出来的语音胡言乱语，不开vllm正常语音

### ISSUE THREAD

**Issue by worm128** [FunAudioLLM/CosyVoice#1601]
# pip list|grep transformer
transformers                             4.51.3

# pip list|grep vllm
vllm                                     0.9.0

我启动的是cosyvoice2的模型CosyVoice2-0.5B，启动和合成没有保存，但是语音发音是乱的。
CosyVoice2(args.model_dir, load_jit=True, load_trt=True, load_vllm=True, fp16=True)
我已经按照官方的版本来安装，发现合成出来还是语音混乱的
看了几个帖子也是一样问题：
#1481 

我的py依赖包安装版本：
Package                                  Version
---------------------------------------- -------------
absl-py                                  2.3.1
aiofiles                                 23.2.1
aiohappyeyeballs                         2.6.1
aiohttp                                  3.12.15
aiosignal                                1.4.0
airportsdata                             20250909
aliyun-python-sdk-core                   2.16.0
aliyun-python-sdk-kms                    2.16.5
annotated-types                          0.7.0
antlr4-python3-runtime                   4.9.3
anyio                                    4.11.0
appdirs                                  1.4.4
APScheduler                              3.11.0
astor                                    0.8.1
async-timeout                            5.0.1
attrs                                    25.3.0
audioread                                3.0.1
beautifulsoup4                           4.13.5
blake3                                   1.0.6
cachetools                               5.5.2
certifi                                  2025.8.3
cffi                                     2.0.0
charset-normalizer                       3.4.3
click                                    8.3.0
cloudpickle                              3.1.1
coloredlogs                              15.0.1
compressed-tensors                       0.9.4
conformer                                0.3.2
contourpy                                1.3.2
crcmod                                   1.7
cryptography                             46.0.2
cupy-cuda12x                             13.6.0
cycler                                   0.12.1
Cython                                   3.1.4
decorator                                5.2.1
depyf                                    0.18.0
diffusers                                0.29.0
dill                                     0.4.0
diskcache                                5.6.3
distro                                   1.9.0
dnspython                                2.8.0
editdistance                             0.8.1
einops                                   0.8.1
email-validator                          2.3.0
exceptiongroup                           1.3.0
fastapi                                  0.115.6
fastapi-cli                              0.0.13
fastapi-cloud-cli                        0.2.1
fastrlock                                0.8.3
ffmpy                                    0.6.1
filelock                                 3.19.1
flashinfer-python                        0.3.1
flatbuffers                              25.9.23
fonttools                                4.60.0
frozenlist                               1.7.0
fsspec                                   2024.12.0
funasr                                   1.2.7
gdown                                    5.1.0
gguf                                     0.17.1
google-auth                              2.40.3
google-auth-oauthlib                     1.0.0
googleapis-common-protos                 1.70.0
gradio                                   5.23.3
gradio_client                            1.8.0
groovy                                   0.1.2
grpcio                                   1.63.2
grpcio-tools                             1.6.0
h11                                      0.16.0
hf-xet                                   1.1.10
hjson                                    3.1.0
httpcore                                 1.0.9
httptools                                0.6.4
httpx                                    0.28.1
huggingface-hub                          0.35.1
humanfriendly                            10.0
hydra-core                               1.3.2
HyperPyYAML                              1.2.2
idna                                     3.10
importlib_metadata                       8.7.0
importlib_resources                      6.5.2
inflect                                  7.3.1
interegular                              0.3.3
jaconv                                   0.4.0
jamo                                     0.4.1
jieba                                    0.42.1
Jinja2                                   3.1.6
jiter                                    0.11.0
jmespath                                 0.10.0
joblib                                   1.5.2
jsonschema                               4.25.1
jsonschema-specifications                2025.9.1
kaldiio                                  2.18.1
kiwisolver                               1.4.9
lameenc                                  1.8.1
lark                                     1.2.2
lazy_loader                              0.4
librosa                                  0.10.2
lightning                                2.2.4
lightning-utilities                      0.15.2
llguidance                               0.7.30
llvmlite                                 0.44.0
lm-format-enforcer                       0.10.12
Mako                                     1.3.3
Markdown                                 3.9
markdown-it-py                           4.0.0
MarkupSafe                               2.1.5
matplotlib                               3.7.5
mdurl                                    0.1.2
mistral_common                           1.8.5
modelscope                               1.20.0
more-itertools                           10.8.0
mpmath                                   1.3.0
msgpack                                  1.1.1
msgspec                                  0.19.0
multidict                                6.6.4
nest-asyncio                             1.6.0
networkx                                 3.1
ninja                                    1.13.0
numba                                    0.61.2
numpy                                    1.26.4
nvidia-cublas-cu12                       12.6.4.1
nvidia-cuda-cupti-cu12                   12.6.80
nvidia-cuda-nvrtc-cu12                   12.6.77
nvidia-cuda-runtime-cu12                 12.6.77
nvidia-cudnn-cu12                        9.5.1.17
nvidia-cudnn-frontend                    1.14.1
nvidia-cufft-cu12                        11.3.0.4
nvidia-cufile-cu12                       1.11.1.6
nvidia-curand-cu12                       10.3.7.77
nvidia-cusolver-cu12                     11.7.1.2
nvidia-cusparse-cu12                     12.5.4.2
nvidia-cusparselt-cu12                   0.6.3
nvidia-ml-py                             13.580.82
nvidia-nccl-cu12                         2.26.2
nvidia-nvjitlink-cu12                    12.6.85
nvidia-nvtx-cu12                         12.6.77
oauthlib                                 3.3.1
omegaconf                                2.3.0
onnx                                     1.16.0
onnxruntime-gpu                          1.18.1
openai                                   1.109.1
openai-whisper                           20250625
opencv-python-headless                   4.11.0.86
opentelemetry-api                        1.37.0
opentelemetry-exporter-otlp              1.37.0
opentelemetry-exporter-otlp-proto-common 1.37.0
opentelemetry-exporter-otlp-proto-grpc   1.37.0
opentelemetry-exporter-otlp-proto-http   1.37.0
opentelemetry-proto                      1.37.0
opentelemetry-sdk                        1.37.0
opentelemetry-semantic-conventions       0.58b0
opentelemetry-semantic-conventions-ai    0.4.13
orjson                                   3.11.3
oss2                                     2.19.1
outlines                                 0.1.11
outlines_core                            0.1.26
packaging                                24.2
pandas                                   2.3.2
partial-json-parser                      0.2.1.1.post6
pillow                                   10.3.0
pip                                      24.0
platformdirs                             4.2.1
polygraphy                               0.49.10
pooch                                    1.8.2
prometheus_client                        0.23.1
prometheus-fastapi-instrumentator        7.1.0
propcache                                0.3.2
protobuf                                 5.26.0
psutil                                   7.1.0
py-cpuinfo                               9.0.0
pyarrow                                  18.1.0
pyasn1                                   0.6.1
pyasn1_modules                           0.4.2
pycountry                                24.6.1
pycparser                                2.23
pycryptodome                             3.23.0
pycuda                                   2024.1
pydantic                                 2.11.7
pydantic_core                            2.33.2
pydantic-extra-types                     2.10.5
pydub                                    0.25.1
Pygments                                 2.19.2
pynini                                   2.1.5
pynndescent                              0.5.13
pynvml                                   13.0.1
pyparsing                                3.2.5
PySocks                                  1.7.1
python-dateutil                          2.9.0.post0
python-dotenv                            1.1.1
python-json-logger                       3.3.0
python-multipart                         0.0.20
pytools                                  2024.1.2
pytorch-lightning                        2.5.5
pytorch-wpe                              0.0.1
pytz                                     2025.2
pyworld                                  0.3.4
PyYAML                                   6.0.3
pyzmq                                    27.1.0
ray                                      2.49.2
referencing                              0.36.2
regex                                    2025.9.18
requests                                 2.32.5
requests-oauthlib                        2.0.0
rich                                     13.7.1
rich-toolkit                             0.15.1
rignore                                  0.6.4
rpds-py                                  0.27.1
rsa                                      4.9.1
ruamel.yaml                              0.18.15
ruamel.yaml.clib                         0.2.14
ruff                                     0.13.2
safehttpx                                0.1.6
safetensors                              0.6.2
scikit-learn                             1.7.2
scipy                                    1.15.3
semantic-version                         2.10.0
sentencepiece                            0.2.1
sentry-sdk                               2.39.0
setuptools                               80.9.0
shellingham                              1.5.4
six                                      1.17.0
sniffio                                  1.3.1
soundfile                                0.12.1
soupsieve                                2.8
soxr                                     1.0.0
starlette                                0.41.3
sympy                                    1.14.0
tabulate                                 0.9.0
tensorboard                              2.14.0
tensorboard-data-server                  0.7.2
tensorboardX                             2.6.4
tensorrt-cu12                            10.0.1
tensorrt-cu12-bindings                   10.0.1
tensorrt-cu12-libs                       10.0.1
threadpoolctl                            3.6.0
tiktoken                                 0.11.0
tokenizers                               0.21.4
tomlkit                                  0.13.3
torch                                    2.7.0
torch-complex                            0.4.4
torchaudio                               2.7.0
torchmetrics                             1.8.2
torchvision                              0.22.0
tqdm                                     4.67.1
transformers                             4.51.3
triton                                   3.3.0
ttsfrd                                   0.4.2
ttsfrd-dependency                        0.1
typeguard                                4.4.4
typer                                    0.19.2
typing_extensions                        4.15.0
typing-inspection                        0.4.1
tzdata                                   2025.2
tzlocal                                  5.3.1
umap-learn                               0.5.9.post2
urllib3                                  2.5.0
uvicorn                                  0.30.0
uvloop                                   0.21.0
vllm                                     0.9.0
watchfiles                               1.1.0
websockets                               15.0.1
Werkzeug                                 3.1.3
WeTextProcessing                         1.0.3
wget                                     3.2
xformers                                 0.0.30
xgrammar                                 0.1.19
yarl                                     1.20.1
zipp                                     3.23.0
